doc/command_ref.rst

```diff
@@ -979,15 +979,30 @@ Module subcommands take :ref:`\<module-spec>\ <specifying_modules-label>`... arg
     In case no profile was provided, all default profiles get installed.
     Module streams get enabled accordingly.
 
-    This command cannot be used for switching module streams. It is recommended to remove all
-    installed content from the module and reset the module using the
-    :ref:`reset <module_reset_command-label>` command. After you reset the module, you can install
-    the other stream.
+    This command cannot be used for switching module streams. It is recommended to use the
+    :ref:`dnf module switch <module_switch_command-label>` command.
 
 ``dnf [options] module update <module-spec>...``
     Update packages associated with an active module stream, optionally restricted to a profile.
     If the `profile_name` is provided, only the packages referenced by that profile will be updated.
 
+.. _module_switch_command-label:
+
+``dnf [options] module switch <module-spec>...``
+    Switch or enable module stream, change a version of installed packages to a version provided by
+    the new stream, and remove packages from the old stream that are no longer available. It also
+    updates installed profiles if they are available for the new stream. In case that <module-spec>
+    contains profile, it installs that profile and does not update already installed profiles.
+
+    It could be used as a stronger :ref:`dnf module enable <module_enable_command-label>` command
+    when it doesn't only enable modules, but also it makes distrosync to all modular packages in
+    enabled modules.
+
+    It could be also used as a stronger :ref:`dnf module install <module_install_command-label>`
+    command, but it requires to specify profiles that are supposed to be installed, because `switch`
+    command do not use `default profiles`. The `switch` command doesn't only install profiles, but
+    also it makes distrosync to all modular packages in installed module.
```

Comment:
> It **can also be** used as a stronger :ref:`dnf module install <module_install_command-label>` command, but it requires to specify profiles that are supposed to be installed, because **the** `switch` command **does** not use `default profiles`. The `switch` command doesn't only install profiles, **it also** makes **a `distrosync`** to all modular packages in **the** installed module.

Note I put the `distrosync` in  backticks to make it clear it's a "command" (or a specific operation).